syntax = "proto3";
package bookandride;
message Book {
 int32 id = 1;
 string title = 2;
 string author = 3;
 double price = 4;
 bool in_stock = 5;
}
